commands/create/lib/create.js

```diff
@@ -136,7 +136,8 @@ const create = async (argv) => {
           fse.ensureDirSync(path.resolve(destination, next));
           traverse(p, next);
         } else if (file[0] === '_') {
-          const newFileName = file === '_package.json' ? 'package.json' : `.${file.substr(1)}`;
+          const configFiles = ['_package.json', '_playwright.config.js'];
```

Comment:
The playwright config file isn't edited, right? so I don't think you want/need the `_` here. I think the idea here is that the `_package.json` is a template that gets injected with the current versions of Nebula, but the rest are just raw files.

Reply:
Yes, you are correct. 
the `playwright.config.js` file will be copied over without any issue. 

✅ fixed!